nodejs/week2/app.js

```diff
@@ -0,0 +1,68 @@
+import express from "express";
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.use(express.json());
+
+// import json to app.js with ES6
+import { readFile } from "fs/promises";
```

Comment:
You can actually import a file pretty easily with ```import json from `./document.json` assert { type: `json` }```

Reply:
There is a problem that I cant import this code, I have asked mentor last week but it cant be fixed 
so I will remain the original one :)